sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/Azure.IoT.DeviceUpdate.csproj

```diff
@@ -1,22 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>Device Update for IoT Hub client library</AssemblyTitle>
-    <Version>1.0.0-beta.3</Version>
+    <Version>1.0.0-beta.1</Version>
     <Description>
       This is the Device Update for IoT Hub client library for developing .NET applications with rich experience.
     </Description>
     <PackageTags>Azure Device Update</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <!-- These supressions should be removed in a production library -->
-    <NoWarn>$(NoWarn);CS1591;AZC0001;AZC0012</NoWarn>
+    <NoWarn>$(NoWarn);CS0169;AZC0016;AZC0012</NoWarn>
```

Comment:
It is not needed for this preview, but we will need to remove this warning suppression before the library GA's.

Reply:
I will file an issue to track all those requirement for GA.

Reply:
Would you please add the link to that issue here?

Reply:
The issue to track: https://github.com/Azure/azure-sdk-for-net/issues/26455